.github/workflows/go.yaml

```diff
@@ -22,6 +22,8 @@ jobs:
 
     # Run `go build` to cache things for the linter, so it doesn't time out.
     - run: go build -v ./...
+      env:
+        CGO_ENABLED: '0'
```

Comment:
Nit: I don't think it needs the single quotes.

Reply:
It's not totally necessary, but makes my schema validator happier (because `env` is a `map[string]string`, and `0` in yaml is a number).  GitHub would internally convert it to a string either way.